REVIEW.md

```diff
@@ -0,0 +1,257 @@
+proposal for a new feature), but these are the themes to be addressed.
+
+- [ ] Motivation 
+    - Do we want this? 
+    - Would it help multiple users to have this accepted into Spyder?
+    - Does it help to solve bugs? 
+    - Is it a necessary refactoring?
+
+- [ ] Tests
+    - Do the tests pass in the continuous integration build? If appropriate, 
+    help the contributor understand why tests failed.
+    - Are regression tests needed? If a reproducible bug is being fixed in 
+    this PR, it may be needed and reviewers can help the contributor design 
+    and implement them. In case the contributor is unable to add the test, the 
+    reviewer should add it in a follow-up PR.
+ 
+- [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy? 
+    - Should variable names be improved for clarity or consistency? 
+    - Should comments or docstrings be added? 
+    - Should comments be removed as unhelpful or extraneous? 
+    - Will the cost of maintaining a new feature be worth its benefits? 
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Does the user experience match existing patterns in Spyder? If it does 
+    not, why was the choice made?
+    - Is in-interface text (new or modified) appropriately descriptive and 
+    free of spelling or grammar issues that would make its meaning unclear?
+
+Review that addresses these questions on a PR will be prefixed with 
+`[required]` so that the PR author(s) are clear that they are blockers. PR 
+authors are responsible for addressing these reviews if prompted by the 
+reviewer in order for that PR to be eligible for merging.
+
+### Optional review
+
+The following are themes that may be helpful to cover on a PR, but are optional. 
+
+**Note**: The difference between an objective improvement and a subjective 
+preference isn’t always clear. Reviewers should recall that code review is 
+primarily about reducing risk in the project. When reviewing code, one should 
+aim at preventing situations which may require a bug fix, a deprecation, or a 
+retraction.
+
+- [ ] Improvements
+    - Could the code easily be rewritten to run much more efficiently for 
+    relevant settings?
+    - Is the code backwards compatible with previous versions? (Or is a 
+    deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues 
+    surrounding this change? Are there non-urgent concerns that the team 
+    should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added? Are there images that should be 
+    resized or optimized in another way?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns 
+    in Spyder? If it does not, is there a reason?
```

Comment:
Right now, this line is in both [required] and [optional] sections. I don't know where reviewers would like this to actually fall in terms of priorities.

Some thoughts:
- Having consistent UI patterns does help users navigate and use software more comfortably.
- Many UI choices will be made through Qt. Qt can be a difficult thing to manipulate for many contributors and may also be difficult for reviewers to guide contributors to.

Reply:
@ccordoba12 @dalthviz @steff456 Thoughts?

Reply:
I think it should be only in optional, like sometimes the effort to match in Qt may be really big, and sometimes it may be small. Which means that if something is really hard to implement in Qt maybe we can add a followup issue to try to not loose track of it.

Reply:
I think this was resolved by moving this, aside from what appears to be an unintended duplication of this here again (see [suggestion](https://github.com/spyder-ide/spyder/pull/16789/files#r755547640)) perhaps due to the merge conflict?